README.md

```diff
@@ -75,6 +50,7 @@ const createPaymentSession = async (amount, currency, reason) => {
     reason: data.reason,
     otp: data.otp,
     paymentShortReference: data.shortReference,
+    displayName: data.displayName,
```

Comment:
extra comma?

Reply:
in our code style we use extra comma on the last property in every object